cmd/scan.go

```diff
@@ -71,6 +71,7 @@ func defineScanFlags(cmd *cobra.Command) {
 	scanCmd.Flags().Bool("disable-flow-separation-by-data-element", false, "Experimental: If specified, filtering of flow using 'flow separation by data element algorithm' will be avoided")
 	scanCmd.Flags().Bool("disable-2nd-level-closure", false, "Experimental: If specified, 2nd level source derivation will be turned on")
 	scanCmd.Flags().Bool("generate-unresolved-name-report", false, "Flag to enable generation unresolved method name reports")
+	scanCmd.Flags().Bool("test-output", false, "Flag to generate unfiltered flow output getting direclty from joern")
```

Comment:
Please modify the description to:
```
If specified, additionally generates an unfiltered flow report
```

Also, let's rename the flag to `generate-unfiltered-report`.

Reply:
Done, changed to `generate-unfiltered-report` PTAL @ojaswa1942